PUMI/pipelines/func/func2standard.py

```diff
@@ -23,30 +25,32 @@ def qc(wf, **kwargs):
     wf.connect(qc_vol2png, 'out_file', 'sinker', wf.name)
 
 
-@FuncPipeline(inputspec_fields=['anat', 'linear_reg_mtrx', 'nonlinear_reg_mtrx', 'func', 'reference_brain'],
+@FuncPipeline(inputspec_fields=['anat', 'linear_reg_mtrx', 'nonlinear_reg_mtrx', 'func', 'reference_brain',
+                                'bbr2ants_source_file'],
               outputspec_fields=['out_file'])
-def func2standard(wf, func_is_3D=True, stdreg='ants', interp="NearestNeighbor", **kwargs):
+def func2standard(wf, stdreg='ants', interp='NearestNeighbor', **kwargs):
```

Comment:
why NearestNeighbour? Is this used to transform a mask?
Please double check that the 4d functional images are converted to std space with something better than NN.